src/+otp/+constrainedpendulum/constraintsjacobian.m

```diff
@@ -0,0 +1,27 @@
+function dc = constraintsjacobian(~, state, g, m, ~, ~)
```

Comment:
What Jacobian is this exactly?  For the DAE
y' = f(y, z)
0 = g(z)
is it g_z or f_z*g_z

Reply:
in this formulation. y' = f(y) - (g_y)^T z. T is transpose. 

so g_y*f_z = g_y*((g_y)^T)

Reply:
this jacobian is actually just g_y